plugins/database/src/db_plugin.cpp

```diff
@@ -11477,6 +11495,41 @@ irods::error db_calc_usage_and_quota_op(
 #    define IRODS_QUOTA_CAST_TYPE_AS " AS BIGINT "
 #endif
 
+    /* This query sums up data object sizes and groups them by resource and user.
+       This gets a total usage per resource for each user. This is inserted into
+       R_QUOTA_USAGE. Additionally, after adding the total storage resource
+       usage for each user in the base case, it will recursively add
+       more rows with the same usage for each leaf resource's parent,
```

Comment:
since it walks up to the parents of coordinating resources as well...

```suggestion
       more rows with the same usage for each resource's parent,
```